### ISSUE TITLE
Not pulling data

### ISSUE THREAD

**Issue by cheeseheadnate** [AnthoneyKalasho/MlbTheShow18MarketAssistant#2]
When I run your program it doesn't grab any updates ... "updating page 0 of 0". Not sure whats going on I have everything installed properly.

**Comment by AnthoneyKalasho** [comment]
Yeah unfortunately the card data got put behind a login wall which broke the program for now. In the mean time I put together this userscript that has a similar function. 

https://greasyfork.org/en/scripts/40060-mlb-the-show-nation-profit-calculator

Also take a look at these userscripts made by sreyemnayr

https://greasyfork.org/en/scripts?set=16070

**Comment by cheeseheadnate** [comment]
awesome thanks Anthoney.

**Comment by DarkBugz** [comment]
I can't get the scripts to work either. not yours or sreyemnayr's. they say running but don't actually do anything
